### PR TITLE
Do not assume font handle is present in assets.

### PR DESCRIPTION
**Pull request by Bobox214**
This is a tentative fix for #489 

Code changed to not do anything if the font handle is not yet present in the assets<fonts> resource, removing the panicking 'unwraps'.

Not sure if this is the correct, as there could a design assumption broken before arriving to this pieces of code, and I am not yet fluent enough in bevy to know that.

This fixes both testcases ( late loading of font, or referencing inexistent fonts).

### CODE REVIEW

**Comment by Bobox214** [comment]
Should I also propose  a CHANGELOG.md modification ?

I wonder if this is the correct way to do, adding lines in changelog in the PR themselves, as this will surely creates conflicts when two PR are merged that both add lines changelog. Needing a rebase and merge fix for the second applied one.

**Comment by memoryruins** [comment]
> Should I also propose a CHANGELOG.md modification ?

If you would like, I think it's okay, but that's a good point about merge conflicts. I wouldn't mind adding this to the changelog after the fix merges. I currently have a branch with some changelog updates https://github.com/memoryruins/bevy/blob/changes/CHANGELOG.md. It's a few days out of date, but I'll update it and open a PR with it soon.

**Comment by cart** [comment]
This also fixes #112.

Changelog policy is still TBD as we find out what works. For now, lets make it a separate change.

**Review by cart — APPROVED** [review]
This is good to go once merge conflicts are resolved

**Comment by Bobox214** [comment]
Merge conflicts have been resolved.

File: crates/bevy_text/src/font_atlas_set.rs
Comment (TheNeikos) [inline crates/bevy_text/src/font_atlas_set.rs:56]: As an alternative, you could make an early return here in case the font doesn't exist yet. This way the rest of the function isn't indented.

File: crates/bevy_ui/src/widget/text.rs
Comment (TheNeikos) [inline crates/bevy_ui/src/widget/text.rs:64]: Similarly here, a `continue` would have the same effect without indenting the whole block

File: crates/bevy_text/src/font_atlas_set.rs
Comment (cart) [reply crates/bevy_text/src/font_atlas_set.rs:56]: I think that would require this:
```rust
if fonts.get(&self.font).is_some() { return; }
let font = self.fonts.get(&self.font).unwrap();
```

Which is both a double hash, slightly less safe, and less idiomatic. I definitely love flattening out indentation, but im not sure thats the right call here.

Comment (TheNeikos) [reply crates/bevy_text/src/font_atlas_set.rs:56]: A simple way is:

```rust
let font = if let Some(font) = fonts.get(&self.font) { font } else { return };
```

**Comment by cart** [comment]
I'm merging this now as I think its important for the 0.2.0 release happening today. @TheNeikos I'll address your code style comments in a follow up pr.

**Pull request merged**
